app/controllers/katello/api/v2/products_bulk_actions_controller.rb

```diff
@@ -35,7 +35,7 @@ def sync_products
       syncable_roots = RootRepository.where(:product_id => syncable_products).has_url
 
       syncable_roots = syncable_roots.yum_type if skip_metadata_check || validate_contents
-      syncable_roots = syncable_roots.where.not(:download_policy => ::Runcible::Models::YumImporter::DOWNLOAD_ON_DEMAND) if validate_contents
+      syncable_roots = syncable_roots.where.not(:download_policy => 'on_demand') if validate_contents
```

Comment:
Should we put these constants in somewhere like Katello::RootRepository ?

Reply:
+1 to removing runcible references though!

Reply:
Sure, I can do that.